test/lib/game/round_test.exs

```diff
@@ -0,0 +1,17 @@
+defmodule RoundTest do
+  use ExUnit.Case, async: true
+
+  test "start_link/1 takes users and sets state" do
+    users = ["Tim", "Eli"]
+    {:ok, pid} = Reactor.Game.Round.start_link(users)
+    assert is_pid(pid)
```

Comment:
you're literally just testing a implementation of gen server at that point though. if you really want to make sure state is set as you'd expect, then you need to expose it via a function